includes/manage.inc

```diff
@@ -38,15 +40,42 @@ function islandora_simple_workflow_manage_form($form, $form_state) {
     );
   }
   else {
+    $render_parents = variable_get('islandora_simple_workflow_toggle_parent_column', FALSE);
```

Comment:
Is there a need for not showing the new parent column? This is a great improvement but a toggle means documentation